toolkit/tools/imagecustomizerapi/service.go

```diff
@@ -7,33 +7,29 @@ import (
 	"fmt"
 )
 
-type Service struct {
-	Name string `yaml:"Name"`
-}
-
-func (s *Service) IsValid() error {
-	if s.Name == "" {
+func serviceNameIsValid(name string) error {
+	if name == "" {
 		return fmt.Errorf("name of service may not be empty")
 	}
 
 	return nil
 }
 
 type Services struct {
-	Enable  []Service `yaml:"Enable"`
-	Disable []Service `yaml:"Disable"`
+	Enable  []string `yaml:"Enable"`
+	Disable []string `yaml:"Disable"`
 }
 
 func (s *Services) IsValid() error {
 	for i, service := range s.Enable {
-		if err := service.IsValid(); err != nil {
-			return fmt.Errorf("invalid service '%s' in Service.Enable at index %d: %w", service.Name, i, err)
+		if err := serviceNameIsValid(service); err != nil {
+			return fmt.Errorf("invalid service '%s' in Service.Enable at index %d: %w", service, i, err)
```

Comment:
nit: could you enclose variable value in `()` instead of `''`when returning err.
`return fmt.Errorf("invalid service (%s) in Service.Enable at index (%d): %w", service, i, err)`

Reply:
Fixed